documentation/sphinx/source/class-scheduling.rst

```diff
@@ -106,7 +106,13 @@ Subspaces have a :meth:`pack` method for defining keys. To store the records for
 Transactions
 ------------
 
-We're going to rely on the powerful guarantees of transactions to help keep all of our modifications straight, so let's look at a nice way that the FoundationDB Python API lets you write a transactional function. By using a decorator, an entire function is wrapped in a transaction. Let's write the very simple ``add_class`` function we will use to populate the database's class list::
+We're going to rely on the powerful guarantees of transactions to help keep all of our modifications straight.
+
+It is important to note that transactions will not timeout by default. You should set a timeout (adjust from 3000ms to fit your needs) with::
+
+    tr.options.set_timeout(3000)
```

Comment:
This seems like maybe the wrong place for this (especially as the other methods in the tutorial don't set the timeout. It might make sense to put this after the explanation of what the retry loops look like. (After line 134 in the old docs; after line 140 after this change.)

Reply:
Seems valid to me. Moved to the end of the transactions section